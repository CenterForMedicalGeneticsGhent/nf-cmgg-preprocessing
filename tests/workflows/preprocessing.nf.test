nextflow_workflow {

    name "Test Workflow PREPROCESSING"
    script "workflows/preprocessing.nf"
    workflow "PREPROCESSING"

    tag "workflows"
    tag "workflows/preprocessing"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            workflow {
                """
                // define inputs of the workflow here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            assert workflow.success
            assert snapshot(workflow.out).match()
        }

    }

}
