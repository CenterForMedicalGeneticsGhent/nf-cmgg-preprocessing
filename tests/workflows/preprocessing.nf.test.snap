{
    "preprocessing - fastq - bwa - bamsormadup - roi": {
        "content": [
            "multiqc_report.html",
            [
                "versions.yml:md5,13e87ed415a2e15f5ee6cd61a696f159",
                "versions.yml:md5,1752aa75c91d2f92093dac3af3c392f1",
                "versions.yml:md5,2789454a6d07582f30cca7623d431a2b",
                "versions.yml:md5,321e55c8f19102dc87a10e981635edda",
                "versions.yml:md5,4257181c63c732a41de1cb83471b289b",
                "versions.yml:md5,4cadd59c9520ebd940030a4cc6d39d64",
                "versions.yml:md5,a3854dfc77096728996132543bfa9c39",
                "versions.yml:md5,ca47a131058059f9f6c12ba994a3b5ab",
                "versions.yml:md5,d11c133ecb39ba9f6d7e081a8a6ff868",
                "versions.yml:md5,fcbb82b85126720b43a2f8380bcf2f81"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-26T12:30:12.605008"
    },
    "preprocessing - fastq - bwa - bamsormadup - roi - no coverage/no picard": {
        "content": [
            "multiqc_report.html",
            [
                "versions.yml:md5,13e87ed415a2e15f5ee6cd61a696f159",
                "versions.yml:md5,2789454a6d07582f30cca7623d431a2b",
                "versions.yml:md5,321e55c8f19102dc87a10e981635edda",
                "versions.yml:md5,4257181c63c732a41de1cb83471b289b",
                "versions.yml:md5,d11c133ecb39ba9f6d7e081a8a6ff868",
                "versions.yml:md5,fcbb82b85126720b43a2f8380bcf2f81"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-26T12:33:36.468109"
    },
    "preprocessing - fastq - bwa - bamsormadup - no roi": {
        "content": [
            "multiqc_report.html",
            [
                "versions.yml:md5,13e87ed415a2e15f5ee6cd61a696f159",
                "versions.yml:md5,1752aa75c91d2f92093dac3af3c392f1",
                "versions.yml:md5,2789454a6d07582f30cca7623d431a2b",
                "versions.yml:md5,321e55c8f19102dc87a10e981635edda",
                "versions.yml:md5,4257181c63c732a41de1cb83471b289b",
                "versions.yml:md5,4cadd59c9520ebd940030a4cc6d39d64",
                "versions.yml:md5,ca47a131058059f9f6c12ba994a3b5ab",
                "versions.yml:md5,d11c133ecb39ba9f6d7e081a8a6ff868",
                "versions.yml:md5,f86ff6a955bce1589780c33ee28a1126",
                "versions.yml:md5,fcbb82b85126720b43a2f8380bcf2f81"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-26T12:32:26.471388"
    }
}